src/main/java/com/es/phoneshop/model/order/Order.java

```diff
@@ -13,10 +13,9 @@
 
 @Getter
 @Setter
-public class Order extends Cart implements Entity {
+public class Order extends Cart implements Entity<String> {
 
-    private Long id;
-    private String secureId;
+    private String id;
```

Comment:
still don't get why we can't leave long as id
you generate the value via UUID, it's a random value, no matter if it's long or string
in all previous pojo-s your id was long